traj/scripts/demo

```diff
@@ -66,16 +73,17 @@ max_jerks = np.deg2rad(np.array([
     8000.0,
     8000.0,
     16000.0,
-]))
-
-joint_names = ['joint_1', 'joint_2', 'joint_3', 'joint_4', 'joint_5', 'joint_6']
+])) * 1.0
 
 rospy.init_node('traj_demo')
 follow_trajectory_client = actionlib.SimpleActionClient('joint_trajectory_action', FollowJointTrajectoryAction)
 follow_trajectory_client.wait_for_server()
 rospy.loginfo('Connected to joint trajectory server')
 
 plot_trajectory = rospy.get_param('~plot', False)
+joint_prefix = rospy.get_param('~joint_prefix', '')
+
+joint_names = [joint_prefix + name for name in 'joint_1', 'joint_2', 'joint_3', 'joint_4', 'joint_5', 'joint_6']
```

Comment:
Is this Python `3.7` syntax? It errors out for me on `3.5`.

Reply:
It works in Python 2.7. I think that python is probably interpreting it as a tuple. It's unclear though; i'll add braces. (Are you using python 3.x with ROS?)

Reply:
Hm, weird.

I'll try again.

> Are you using python 3.x with ROS?

Not in this workspace, but it's actually not that difficult. See [how to define ROS kinetic to use python3 instead of python2.7?](https://answers.ros.org/question/237613) fi (both Dirk's and my answer).

Reply:
Yeah, i know it's possible with a virtualenv. I have a bad habit of not using virtualenv though. I've been using clion even for python code lately though, and it has builtin virtualenv support, so I don't really have an excuse anymore.

Reply:
Brackets added